code/modules/client/preferences.dm

```diff
@@ -112,6 +112,15 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	var/action_buttons_screen_locs = list()
 
+	/// Agendered spessmen can choose whether to have a male or female bodytype

```

Comment:
thanks for this, but, uh, why was this rolled into this wounds PR instead of being put in its own PR?

Reply:
Because it's a small physiology change that takes place in the same sections of code I'm already working in